synapse/config/server.py

```diff
@@ -351,6 +375,20 @@ def default_config(self, server_name, data_dir_path, **kwargs):
         #  - nyc.example.com
         #  - syd.example.com
 
+        # Prevent federation requests from being sent to the following
+        # blacklist IP address CIDR ranges.
```

Comment:
please could you explain what the default is (even if it's that nothing is blacklisted)?